openedx/adg/common/mailchimp_pipeline/tasks.py

```diff
@@ -0,0 +1,77 @@
+"""
+Tasks for Mailchimp pipeline
+"""
+import logging
+
+from celery.task import task
+from django.conf import settings
+
+from openedx.adg.common.course_meta.models import CourseMeta
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+from .client import MailchimpClient
+from .helpers import get_enrollment_course_names_and_short_ids_by_user
+
+log = logging.getLogger(__name__)
+
+
+@task(routing_key=settings.HIGH_PRIORITY_QUEUE)
+def task_send_user_info_to_mailchimp(instance, **kwargs):  # pylint: disable=unused-argument
+    """
+    Add new user data to Mailchimp (audience) list
```

Comment:
Same, It is not just adding new user's data, it is also updating the existing ones, how about ?
```
Syncs user's data with Mailchimp
```